airflow/contrib/hooks/spark_submit_hook.py

```diff
@@ -288,7 +293,9 @@ def submit(self, application="", **kwargs):
         self._process_spark_submit_log(iter(self._submit_sp.stdout.readline, ''))
         returncode = self._submit_sp.wait()
 
-        if returncode:
+        # returncode is the Kubernetes return code, indicates whether pods were started
```

Comment:
Can we make this comment more clearly? The return code is the exit code of the spark-submit operation, and not kubernetes per se.